featuretools/primitives/standard/transform/time_series/utils.py

```diff
@@ -236,3 +237,16 @@ def apply_rolling_agg_to_series(
             num_nans = min_periods - 1 + gap
         applied_rolled_series.iloc[range(num_nans)] = np.nan
     return applied_rolled_series.values
+
+
+def _apply_gap_for_expanding_primitives(
+    x: Series,
+    gap: Union[int, str],
+) -> Optional[Series]:
+    if isinstance(gap, int):
+        return x.shift(gap)
+    else:
+        raise NotImplementedError(
+            "We currently do not support string offsets for the gap parameter in "
+            "Expanding primitives",
```

Comment:
Minor nitpick, but I would change this wording just a bit:

`String offsets are not supported for the gap parameter in Expanding primitives.`